docs/configuration.rst

```diff
@@ -138,9 +138,7 @@ Depending on your particular deployment you most probably will want to change th
 Advanced settings
 -----------------
 
-The ``PASSWORD_SCHEME`` is the password encryption scheme. You should use the
-default value, unless you are importing password from a separate system and
-want to keep using the old password encryption scheme.
+The ``CREDENTIAL_ROUNDS`` (default: 12) is the number of rounds used by the password hashing scheme. You should use the default value.
```

Comment:
```suggestion
The ``CREDENTIAL_ROUNDS`` (default: 12) is the number of rounds used by the password hashing scheme. The number of rounds can be reduced in case faster authentication is needed or increased when more paranoid hashes are desired.
It normally shouldn't be changed, as the default value is explicitly chosen to be secure while still being fast enough on most systems.
```

I think this should be a little more expressive considering we discussed this very value in matrix :)

Reply:
That's f67c9a9